src/Database/Schema/SqlserverSchema.php

```diff
@@ -12,26 +13,47 @@
  * @since         3.0.0
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
+/**
+ * 
+ * UPDATED VERSION TO WORK WITH CAKEPHP AND MSSQL VIEWS.
+ * 
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
 namespace Cake\Database\Schema;
 
+use Cake\Database\Schema\Table;
+
 /**
  * Schema management/reflection features for SQLServer.
  */
-class SqlserverSchema extends BaseSchema
-{
+class SqlserverSchema extends BaseSchema {
```

Comment:
You need to minimize the noise in this PR, only real changes should be commited, also your IDE seems to be misconfigured.